docs/src/prototypes/mentions/MentionsEditor.tsx

```diff
@@ -82,8 +82,13 @@ const MentionsEditor: React.FunctionComponent<
     dispatch({ type: 'RESET_UPDATE_FLAG' })
   }, [state.shouldUpdate])
 
-  const handleEditorKeyUp = (e: React.KeyboardEvent) => {
-    if (!state.open && e.shiftKey && keyboardKey.getCode(e) === keyboardKey.AtSign) {
+  const handleEditorKeyChange = () => {
+    const selectionContent = window.getSelection().anchorNode.nodeValue
+    if (
+      !state.open &&
+      selectionContent &&
+      selectionContent[window.getSelection().focusOffset - 1] === '@'
+    ) {
```

Comment:
Nit, we can rewrite it to make a bit more readable:

```tsx
    const { anchorNode, focusOffset } = window.getSelection()
    const lastCharacter = anchorNode.nodeValue && anchorNode.nodeValue[focusOffset - 1]

    if (!state.open && lastCharacter === '@') {
```

***

But even current changes should be adjusted, I can't close `Dropdown` by `Escape` key or `Backspace` anymore.

![bug-input](https://user-images.githubusercontent.com/14183168/63340575-ac296880-c347-11e9-96f3-743ce036e63d.gif)

Reply:
It works now 👍